test/libsolidity/InlineAssembly.cpp

```diff
@@ -170,7 +170,7 @@ BOOST_AUTO_TEST_CASE(smoke_test)
 
 BOOST_AUTO_TEST_CASE(surplus_input)
 {
-	CHECK_PARSE_ERROR("{ } { }", ParserError, "Expected token EOS");
+	CHECK_PARSE_ERROR("{ } { }", ParserError, "Expected 'EOS' but got '{'");
```

Comment:
We should explain what `EOS` means.

Reply:
Do you want to change the description for `EOS` in `Token.h`, `Token::friendlyName` or `ParserBase::expectToken` ?

Reply:
I think we can just name it "end of input" or "end of file" in `friendlyName`. We should also have a test for something like `contract a {`.

Reply:
I.e. where the error is `expected ... but got "end of input"`.